src/main/java/org/reactome/release/dataexport/Main.java

```diff
@@ -12,32 +12,40 @@
  * @author jweiser
  */
 public class Main {
-	@Parameter(names={"--overwrite-config-file", "-c"})
-	private boolean overwriteConfigFile;
+	@Parameter(names={"--generate-config-file", "-g"})
+	private boolean generateConfigFile;
```

Comment:
Could you specify the default value explicitly?

Reply:
I can try - I'm not sure if it would interfere with JCommand to specify a default.

Reply:
From what I recall, it shouldn't interfere. In fact, I think specifying defaults is pretty common.

Reply:
Okay, done - it didn't interfere during testing.